DFC.ServiceTaxonomy.GraphSync/Startup.cs

```diff
@@ -126,6 +128,13 @@ public override void Configure(IApplicationBuilder builder, IEndpointRouteBuilde
                 pattern: "Home/Index",
                 defaults: new { controller = "Home", action = "Index" }
             );
+
+            routes.MapAreaControllerRoute(
```

Comment:
Do we need to both map the route and use attribute routing?

Reply:
I'll remove the area route, this was in before the attribute routing